init dataset finished in  1:50:22.784304
saving dataset finished in  0:00:00.035811
Traceback (most recent call last):
  File "/lustre03/project/6066503/yoogo/mywork/keyword_spotting/test.py", line 41, in <module>
    with open('../SpeechCommands/pickle/validation_set.pt', 'wb') as handle:
FileNotFoundError: [Errno 2] No such file or directory: '../SpeechCommands/pickle/validation_set.pt'
